call_apply_bind_exercise/callApplyBind.js

```diff
@@ -0,0 +1,46 @@
+function sumEvenArguments() {
+  var arr = [].slice.call(arguments);
+  var sum = 0;
+  for(var i = 0; i <= arr.length; i++) {
+    if (arr[i] % 2 === 0) {
+      sum += arr[i];
+    }
+  }
+  return sum;
+}
+
+function arrayFrom() {
+  var arr = [].slice.call(arguments);
+  return arr;
+}
+
+function invokeMax(fn, max) {
+  var counter = 0;
+  return function() {
+    var innerArgs = [].slice.call(arguments);
+    if (counter >= max) {
+      return "Maxed Out!";
+    } else {
+      counter++;
+      return fn.apply(this, arguments);
+    }
+  }
+}
+///yeah still need clarification on how this works ^
```

Comment:
Let me know if you want me to go over this with you.  Looks like you go it right.